proto/vtrpc.proto

```diff
@@ -33,8 +33,55 @@ message CallerID {
   string subcomponent = 3;
 }
 
-// ErrorCode is the enum values for Errors
+// ErrorCode is the enum values for Errors. Internally, errors should
+// be created with one of these codes. These will then be translated over the wire
+// by various RPC frameworks.
 enum ErrorCode {
+  SUCCESS = 0;
```

Comment:
I would use the same doc style as a go enum, with one comment per enum too:
// SUCCESS means ...